common-docs/reference/math/random-range.md

```diff
@@ -0,0 +1,15 @@
+# Random Range
+
+Returns a pseudo-random number in the range ``[min, max];`` that is, from ``0`` (inclusive) up to including ``1`` (inclusive), which you can then scale to your desired range. 
```

Comment:
We need to decide if this is a floating point number or integer. I think switching based on the bounds being integer isn't the best design. Maybe rename it randomInt ?